importer/package.json

```diff
@@ -18,6 +28,8 @@
   },
   "dependencies": {
     "avocado": "1.0.0",
-    "replace": "1.1.4"
+    "icon-font-generator": "^2.1.10",
```

Comment:
Has there been, or do we need, a legal review of icon-font-generator, along the lines of OSS checks?

Reply:
Happy to do that, please refer to our conversation on Teams.